draft-ietf-rtcweb-jsep.xml

```diff
@@ -2517,20 +2530,25 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         returned.
           <list style="symbols">
             <t>If the description is of type "offer", and the ICE ufrag or password changed
-              from the previous remote description, [TODO: ref], mark that an ICE restart is needed.</t>
+              from the previous remote description, as described in Section
+            9.1.1.1 of <xref target="RFC5245"/>, mark that an ICE restart is needed.</t>
             <t>Configure the ICE components associated with this media section to use
               the supplied ICE remote ufrag and password for their connectivity checks.</t>
             <t>Pair any supplied ICE candidates with any gathered local candidates, as described
-              in [TODO: ref] and start connectivity checks with the appropriate credentials.</t>
+            in [TODO: ref to <xref target="RFC5245"/> or something else ???]
+            and start connectivity checks with the appropriate credentials.</t>
             <t>If the media section proto value indicates use of RTP:
               <list style="symbols">
                 <t>[TODO: header extensions]</t>
                 <t>For each specified payload type that is also supported by the local implementation,
                   establish a mapping between the
-                  payload type ID and the actual media format. [TODO: ref] If any indicated payload
+                  payload type ID and the actual media format.
+                  [TODO: ref to ??? goes here]
```

Comment:
The 'establish mapping' needs to be explained more - you can just leave this as a TODO for me